railgun/internal/ctrlutils/utils.go

```diff
@@ -141,3 +146,31 @@ func CRDExists(client client.Client, gvr schema.GroupVersionResource) bool {
 	}
 	return true
 }
+
+// retrieve Kong Admin API URL from configured name/namespace service
+func RetrieveKongAdminAPIURL(ctx context.Context, KongAdminAPI string, kubeCfg *rest.Config) (string, error) {
+	namespace, name, err := util.ParseNameNS(KongAdminAPI)
+	if err != nil {
+		return "", fmt.Errorf("failed to pars kong admin api namespace and name. err %v", err)
```

Comment:
There's a typo present, and we can use `%w` to created a [wrapped error](https://golang.org/pkg/fmt/#Errorf) which is more consistent with the rest of the codebase:
```suggestion
		return "", fmt.Errorf("failed to parse kong admin api namespace and name: %w", err)
```